.m2-tmp/com/liferay/com.liferay.gradle.plugins.workspace/1.10.3/com.liferay.gradle.plugins.workspace-1.10.3.pom

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Can you please squash all the `Use latest` and `FAKE GRADLE CACHE` commits into 2 commits? This way it'll be easier for Brian to review the pull 😊